cmd/exo/cmd/privnet_create.go

```diff
@@ -16,11 +17,27 @@ var privnetCreateCmd = &cobra.Command{
 	Short:   "Create private network",
 	Aliases: gCreateAlias,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		name, err := cmd.Flags().GetString("name")
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+		name := args[0]
+		desc, err := cmd.Flags().GetString("description")
 		if err != nil {
 			return err
 		}
-		desc, err := cmd.Flags().GetString("description")
+		startip, err := cmd.Flags().GetString("startip")
+		if err != nil {
+			return err
+		}
+		endip, err := cmd.Flags().GetString("endip")
+		if err != nil {
+			return err
+		}
+		mask, err := cmd.Flags().GetUint("mask")
+		if err != nil {
+			return err
+		}
+		netmask, err := cmd.Flags().GetString("netmask")
```

Comment:
I'd keep only one of them, it's easier to support `255.255.255.0`, `/24` or `24` syntaxes.

Reply:
Ok, now the command supports only netmask (255.255.255.0)